app/scripts/modules/amazon/src/serverGroup/details/serverGroupDetails.aws.controller.js

```diff
@@ -154,6 +154,11 @@ module.exports = angular.module('spinnaker.amazon.serverGroup.details.controller
                 this.changeConfig = {
                   metadata: get(this.serverGroup.entityTags, 'creationMetadata')
                 };
+                if (this.serverGroup.buildInfo.jenkins) {
```

Comment:
Is the `changeConfig` feature specific to Netflix? `buildInfo.jenkins` is not (check out the end of Part 1 of [this codelab](https://www.spinnaker.io/guides/tutorials/codelabs/gce-source-to-prod/#part-2-validate-test)), so I'd guess it would work for anyone using Rosco and having the entity tags feature enabled.